src/test/singlenode_isolation2/expected/bitmap_index_ao_sparse.out

```diff
@@ -30,7 +30,7 @@ COMMIT
 SELECT count(*) FROM ao_sparse WHERE id >= 97 and id <= 99 and gp_segment_id = 0;
```

Comment:
rewrite this query as gp_segment_id = -1?

Reply:
That' better. Updated.